src/plugins/media/nodeviews/mediaNodeView/media.tsx

```diff
@@ -47,12 +52,21 @@ export interface MediaNodeProps extends ReactNodeProps, ImageLoaderProps {
 interface MediaNodeState {
   viewMediaClientConfig?: MediaClientConfig
   contextIdentifierProvider?: ContextIdentifierProvider
+  imageUrl: string
+  imageError: any
 }
 
+const cozyClient = CozyClient.fromDOM()
```

Comment:
not super happy about this, it will get instanced for each image

Reply:
We can’t do that. We should try to have only once instance of CC

Reply:
Agreed but I don't think I will be able to use the one instanced in Editor component. I can't pass props to the plugin, maybe context could work I have to find out

Reply:
Fixed